Corona-Warn-App/src/main/java/de/rki/coronawarnapp/risk/DefaultRiskLevels.kt

```diff
@@ -106,8 +110,21 @@ class DefaultRiskLevels @Inject constructor(
 
         val aggregatedResult = aggregateResults(riskResultsPerWindow)
 
-        return aggregatedResult.totalRiskLevel ==
+        exposureSummaryRepository.exposureWindowEntities = Pair(exposureWindows, aggregatedResult)
```

Comment:
I'm not sure whether its neccessary to persist or "to repo" the exposureWindows... 
On my current point of view, only the Result is needed.

Reply:
renamed it, because it is not a repo anymore - as dicussed